authors/apps/authentication/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase
+
+from authors.apps.authentication.models import (
+    User, UserManager
+)
+
+
+class UserManagerTest(TestCase):
```

Comment:
For this, you should be testing the `User`, not the `UserManager`. 
`UserManager` is used as a custom `objects` in django. This means you should test the `create_user` method like  `User.objects.create_user()`.  Does this make sense??